model/incoming_webhook.go

```diff
@@ -106,6 +331,14 @@ func (o *IncomingWebhook) IsValid() *AppError {
 		return NewAppError("IncomingWebhook.IsValid", "model.incoming_hook.icon_url.app_error", nil, "", http.StatusBadRequest)
 	}
 
+	// TN added
+	if ( ! ( len(o.SecretToken) == 26 || len(o.SecretToken) == 0 )) {
+		return NewAppError("IncomingWebhook.IsValid", "model.incoming_hook.secret_token.app_error", nil, "", http.StatusBadRequest)
+	}
+	//json array of at least 2 ip6 cidr or 5 ip4 cidr 
+	if len(o.WhiteIpList) > 5 {
```

Comment:
Is there a reason I am not grasping for this limitation?

Reply:
@larkox Github has two CIDRs and that perhaps why set the limit, and column size is also limited to that as well.

Reply:
Make sense to have these also as a configurable part? In other words, moving these to a variable so it is more centralized and later can be more easily configured by the user or the MM team?

Reply:
Changed in the new commit.